src/auth/authVerification.js

```diff
@@ -144,7 +164,37 @@ export function verifyAuthRequest(token) {
   })
 }
 
-export function verifyAuthResponse(token, nameLookupURL) {
+/**
+ * Verify the authentication response is valid and
+ * fetch the app manifest file if valid. Otherwise, reject the promise.
+ * @param  {String} token the authentication request token
+ * @return {Promise} that resolves to the app manifest file in JSON format
+ * or rejects if the auth request or app manifest file is invalid
+ * @private
```

Comment:
Should this be marked private if we're going to use this function in blockstack-browser?